pkg/audit/manager.go

```diff
@@ -156,6 +156,12 @@ func (am *Manager) auditManagerLoop(ctx context.Context) {
 
 // Start implements controller.Controller
 func (am *Manager) Start(stop <-chan struct{}) error {
+	if *auditInterval == 0 {
+		log.Info("Audit Manager disabled")
+		<-stop
```

Comment:
not sure if it needs to wait for stop signal here, but doing it to keep the code symmetric 🤷‍♂

Reply:
The way the K8s manager pattern works, when Start() returns the manager tears down the whole process and exits, so would definitely be necessary.

I think it'd be better to only add the Audit Manager to the manager conditionally, to decouple how audit works from whether it's installed. That would mean wrapping the call to manager.Add() in an if statement (those lines I linked to). An example of this is here:

https://github.com/open-policy-agent/gatekeeper/blob/3d5e06b52b4823a0533202e7f444a6ab7a140ecd/pkg/webhook/policy.go#L62-L69